packages/renderer/src/components/ProjectCard/styles.css

```diff
@@ -10,74 +9,88 @@
   text-decoration: none;
   border: 2px solid transparent;
   box-sizing: border-box;
+  background-color: var(--secondary);
+  color: var(--text);
 }
 
-.ProjectCard.has-thumbnail {
-  position: relative;
-  background-size: cover;
-  background-position: center;
+.ProjectCard:hover {
+  border: 2px solid var(--white);
 }
 
-.ProjectCard .project-thumbnail {
-  width: 100%;
+.ProjectCard .video {
+  object-fit: cover;
+}
+
+.ProjectCard .thumbnail {
   background-size: 100% 100%;
```

Comment:
The thumbnails are being stretched by this 100% by 100% size:

![Screenshot 2024-09-09 at 9 49 11 AM](https://github.com/user-attachments/assets/5b82650b-bd87-4f56-9b94-1d9eae073a97)

It would look better if replaced with `background: no-repeat center / cover`:

![Screenshot 2024-09-09 at 9 50 18 AM](https://github.com/user-attachments/assets/b5b3d596-2d1b-4eb0-b5fb-5fc267c28774)

```suggestion
  background: no-repeat center / cover;
```